index.js

```diff
@@ -34,6 +34,7 @@ export function zwitch(key, options) {
    * and it’s result is returned.
    *
    * @param {...unknown} [value]
+   * @this {Options}
```

Comment:
```suggestion
   * @this {unknown}
```

The this value could be anything, as is passed through here. Similar to how all args are passed through!

Reply:
updated